packages/reports/addon/components/common-actions/schedule.js

```diff
@@ -44,16 +45,12 @@ export default Component.extend({
   /**
    * @property {DS.Model} deliveryRule - deliveryRule for the current user
    */
-  deliveryRule: computed('model.deliveryRuleForUser.isFulfilled', function() {
-    return get(this, 'model.deliveryRuleForUser');
-  }),
+  deliveryRule: undefined,
```

Comment:
Could this be:
```js
deliveryRule: reads('model.deliveryRuleForUser')
```

Reply:
I think that would kick off the fetch before opening the schedule modal. We discussed this a while ago and wanted to change the modal to load on opening the modal.

Reply:
It would get kicked off the first time js would do a `get` on it or a hbs would pass it to a component